diff --git a/adh-apis/cmake/Protobuf.cmake b/adh-apis/cmake/Protobuf.cmake
index 8e7d83d..1569b35 100644
--- a/adh-apis/cmake/Protobuf.cmake
+++ b/adh-apis/cmake/Protobuf.cmake
@@ -5,6 +5,14 @@ set(protobuf_MODULE_COMPATIBLE ON)
 find_package(Protobuf CONFIG)
 if(Protobuf_FOUND)
     message(STATUS "Found protobuf via cmake config")
+    # if protobuf-config.cmake is used, we need to explicitly
+    # support the case in which we are cross-compiling and we want
+    # to explicitly specify via CMake command line the protoc to use
+    if(Protobuf_PROTOC_EXECUTABLE)
+      set_target_properties(protobuf::protoc PROPERTIES
+        IMPORTED_LOCATION_RELEASE "${Protobuf_PROTOC_EXECUTABLE}"
+      )
+    endif()
 else()
     message(WARNING "Falling back to cmake FindProtobuf as Protobuf was not found via CONFIG")
     find_package(Protobuf REQUIRED)
