diff --git a/src/protozfits/__init__.py b/src/protozfits/__init__.py
index acc1a14..ed31344 100644
--- a/src/protozfits/__init__.py
+++ b/src/protozfits/__init__.py
@@ -9,7 +9,6 @@ import numpy as np
 from astropy.io import fits
 from google.protobuf.message import Message
 
-from .any_array_to_numpy import any_array_to_numpy
 from .rawzfits import (
     ADH_VERSION_MAJOR,
     ADH_VERSION_MINOR,
@@ -21,6 +20,12 @@ from .version import __version__
 
 ADH_VERSION = f"{ADH_VERSION_MAJOR}.{ADH_VERSION_MINOR}"
 
+
+# this import has to stay here for reasons I really don't understand
+# it fixes an issue where under some circumstances, protobuf modules are
+# imported twice and then errors are raised that classes are not matching
+from .any_array_to_numpy import any_array_to_numpy  # noqa: E402
+
 __all__ = [
     "__version__",
     "ADH_VERSION",
